community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/Tracker.java

```diff
@@ -63,4 +63,7 @@
      * @param value value to set at that index.
      */
     void set( long index, long value );
+
+    @Override
+    void close();
```

Comment:
Ok, if you really want to hide exception why not to make them extend `Resource` instead of `AutoCloseable`?